packages/preview-middleware-client/src/cpe/changes/service.ts

```diff
@@ -248,6 +249,7 @@ export class ChangeService {
      * @returns (event: sap.ui.base.Event) => Promise<void>
      */
     private createOnStackChangeHandler(): (event: Event) => Promise<void> {
+        const handler = modeAndStackChangeHandler(this.sendAction, this.options.rta);
```

Comment:
Does this need a better name like `handleStackChange` so it is a **verb** and not a **noun** like `handler`?

Reply:
renamed